CONTRIBUTING.md

```diff
@@ -295,11 +295,13 @@ To update the version of the Agda spec that the conformance tests are using:
 
 1. Locate the `MAlonzo-code` branch in the [formal-ledger-specifications repo](https://github.com/IntersectMBO/formal-ledger-specifications)
 2. Identify the SHA of the commit that you need, belonging to that branch
-3. In the `cardano-ledger` repository, update the `cabal.project` file by replacing the `tag` and `sha256` fields in the `source-repository-package` stanza with the appropriate values.
+3. In the `cardano-ledger` repository:
+    - Update the `cabal.project` file by replacing the `tag` field in the `source-repository-package` stanza with SHA.
+    - Update the `flake.lock`:
 
-You can determine the correct `sha256` like this:
-  * update the `tag` with the SHA of the chosen commit and run `cabal`. The resulting error message will include the expected `sha256` value.
-  * alternatively, use a tool like `nix-prefetch-git` to fetch and compute the `sha256`
+          ```shell
+          nix flake update formal-ledger-specifications --override-input formal-ledger-specifications github:IntersectMBO/formal-ledger-specifications/SHA
+          ```
```

Comment:
This is great, so much easier!  Had no idea it's possible like this.

I think it would be good to add this alternative instead of completely replacing the old instructions - even though it's more noisy, but it would be good to leave them for non-nix users.

Reply:
I feel the instructions were already nix specific (since after all is haskell.nix that is using the commented out sha256 field).

For non nix users I'd add a comment saying that it is ok to skip this step.

Reply:
This subsection actually belongs to `nix build Infrastructure`.